src/win32.mak

```diff
@@ -65,6 +65,9 @@ OS=windows
 
 ##### Directories
 
+# DM_HOME should be defined in the user's environment
+DM_HOME=Please_define_the__DM_HOME__environment_variable
```

Comment:
I never set DM_HOME (and the required naming does not fit my installation), the C compiler already knows its standard include folders.

Reply:
@rainers what if it's in a non-default location?

Reply:
dmc is not in the default location on my system, but in c:\l\dmc instead. It uses sc.ini, too, containing `INCLUDE="%@P%\..\include";"%@P%\..\mfc\include";%INCLUDE%` which points to the standard include folders.

Reply:
Cool, then can you please submit a small PR to eliminate it entirely? Also it would be great if you also updated the wiki instructions. Thx!

Reply:
See https://github.com/dlang/dmd/pull/7260 
The wiki uses DM_HOME to refer to the DMD installation directory, that seems ok.